python-basic-data-types/escape_seqs.py

```diff
@@ -0,0 +1,4 @@
+print("a\tb")
+print("a\nb")
+print("a\141\x61")
+print("\N{rightwards arrow}")
```

Comment:
If you agree to add the octal and hex examples to the tutorial, please add them here as well.